.github/workflows/style.yaml

```diff
@@ -132,11 +132,6 @@ jobs:
 
       - name: install libyara-dev
         run: |
-          sudo add-apt-repository -n -y "deb http://archive.ubuntu.com/ubuntu/ mantic main restricted universe multiverse"
```

Comment:
We might have to change the runner for this Job to `ubuntu-24.04` until this is aliases to `ubuntu-latest`.